packages/components/src/color-picker/style.native.scss

```
@@ -62,3 +62,26 @@
 .pickerPointer {
 	height: 16px;
 }
+
+.hsv-container {
+	justify-content: center;
+	align-items: center;
+}
+
+.gradient-gontainer {
+	overflow: hidden;
+}
+
+.saturation-slider {
+	top: 0;
+	left: 0;
+	position: absolute;
+	border-color: #fff;
```

Likely doesn't matter at all, as I do not believe anyone is ever going to change the definition of "white," but should we rely upon Sass variables for `#fff`? There is one other instance in this file as well. 
```suggestion
	border-color: $white;
```